### DISCUSSION TITLE
FundOSS Round Deux: Features, Failures and Fixes

### DISCUSSION THREAD

**Discussion by BenJam** [FUNDOSS/fundoss#44] [Programs]
I'm dropping an issue here to compile feature requests, issues that were highlighted and potential fixes. I'll probably break this issue out into other issues once we get into prioritisation:

### Ideas
- open the campaign to all projects (on open collective)
- folks would like closer integration to open collective, specifically they would like donations to register on their collectives as they do with open collective normally.
- folks would like to receive the details of folks that contributed so they can keep them up to date and/or say thank you (may pair well with tighter integration)
- folks wondered whether a pre-paid gift card could be distributed to a team members to vote (and donate) to the campaign 
- grouping projects with others like it, either ahead of schedule (tagging on open collective is a thing) alongside emergent behaviour throughout the round.
- donate outside of the campaign (i.e. contribute on open collective via fundoss)
- Encourage recurring contributions as a part of the campaign
- Provide a link to Open Collective pages on project pages

### Issues
- folks saw matching amounts going down throughout the campaign, leading to them waiting for it to go back up (ohai crypto hangovers) 
- some thought that the format was still too competitive, I wondered whether we might be able to balance that with a more intentional release of funds from the fund over time.
- folks needed to get a copy of their donation receipts after the campaign had closed
- Some folks were a little worried about payment security. Branding the payment gateway provider (strip) and/or adding other third party platforms (Paypal, digital wallets) would increase confidence 
- some projects are requesting receipts for their donations, with company details etc on them. Could we regenerate them with this information on (Open Source Collective can be the org issuing)

**Comment by gusaus** [comment]
Great list @BenJam! If I could add a couple other ideas

- Enable people to donate with stocks (to participants and/or the matching fund) https://blog.opencollective.com/use-your-stock-for-good/
- Provide a way for participants to find contributors (developers, project manager, fundraising, docs, etc.)
- Enable collectives and orgs to use the platform for their own fundraising; for example, large communities like Drupal and WordPress could curate/run their own program; nonprofit or verticals like community media, performing arts, education, etc; location based fundraisers, etc.

Many of these ideas are relevant to the Open Collective product and are similar to what I previously logged here https://github.com/opencollective/opencollective/issues/2232#issuecomment-600810139

**Comment by gusaus** [comment]
This will probably increase donations from the Gitcoin/Bitcoin/Web 3.0. communities for subsequent FundOSS campaigns https://blog.opencollective.com/support-open-source-software-using-crypto/

**Comment by gusaus** [comment]
The [following site](https://giveguide.org/) provides some really nice inspiration for both the FundOSS and Open Collective platforms - here is some functionality 'not' yet listed above.

- Sort filter - 
  - All nonprofits https://giveguide.org/nonprofits
  - Category -  https://giveguide.org/nonprofits/categories
  - Tags - https://giveguide.org/nonprofits?categories=&tags=new,bipoc-led-organization
- Category page - for example https://giveguide.org/categories/creative-expression 
   - donate to multiple orgs and your shopping cart is updated w/o leaving the page. 
   - tally donors, nonprofits, $$ donated. 
   - Display category sponsors - what if there was a way to automate that? maybe could do with the projects feature?
- Project page - for example https://giveguide.org/nonprofits/open-signal
   - allow projects to set funding goal (and display progress) 
   -  add to favorites
   -  display category sponsors

**Comment by gusaus** [comment]
Circling back to this issue as I [mentioned FundOSS as a potential project](https://opencollective.slack.com/archives/C0RD4S62E/p1657835138015829?thread_ts=1656352188.666899&cid=C0RD4S62E) for the OSC Working Group that's taking shape in [#opensource](https://opencollective.slack.com/archives/C0RD4S62E) (Open Collective Slack).

@BenJam mentioned the following when it came up as a potential program - 
> On the funding side, and I think i’ve said this before, I think FundOSS is the wrong format for a fundraising campaign. I don’t like the competitive nature of having a funding pool which is split based on the success of individual projects to campaign well. So that won’t be coming back without some fundamental changes made.

This was my response from the [same Slack discussion](https://opencollective.slack.com/archives/C0RD4S62E/p1659977768387679?thread_ts=1656352188.666899&cid=C0RD4S62E)

> Just will point out the feedback on the competitive nature was provided by the same maintainer/[project](https://fundoss.org/collective/azuracast) that inspired my original call to action (check the very top for a refresher:point_up:). I'm not going to bring Buster or the AzuraCast team members into this discussion (they don't have capacity beyond just keeping the project and themselves sustained) but I can convey their goals and needs as I've been working them (sort of as their wannabe sustainer) in [#openproducer](https://opencollective.slack.com/archives/C22E2D2F2) as well as their very large Discord community.


> Based on pre/post FundOSS discussions, I think AzuraCast, [Kendraio](https://fundoss.org/collective/kendraio) (Kendraio's lead Daniel was the other person providing similar feedback) and other projects I helped onboard all benefited and would participate in future rounds (which seemed to be the plan coming out of the wrap up call and [this issue](https://github.com/FUNDOSS/fundoss/issues/42)). My takeaway from participating in the call and subsequent discussions was that cost and team capacity were the main reasons for the delay. 

To me (especially when considering all the success OSC's partner has had with [Gitcoin Grants](https://gitcoin.co/grants/), we could solve problems with the funding model and contribute to platform improvements by 1) looping in some of the original partners at Gitcoin and 2) engaging community contributors/partners/funders via the working group we're now in the process of forming. :tada:

**Comment by ocdtrekkie** [comment]
Okay, I thought a bit about the good parts and bad parts after talking with and reading the comments from @gusaus and @BenJam.

Things that would be really nice to retain:
1. Exposure opportunities for collectives.
2. Occasional funding from grants secured by OSC and distributed to projects.
3. Quadratic funding helped drive people to make even small donations they might otherwise not because they didn't feel their dollar/five dollars would make a difference.

Things that were problematic to avoid:
1. FundOSS funding participation doesn't roll into OpenCollective funding participation.
2. FundOSS contributors don't get updates/communication from the collective.
4. Maintaining the separate FundOSS platform is expensive/inefficient use of OSC resources.
5. Collectives didn't know if or when future FundOSS rounds would be coming.

My proposal/suggestion: Establish a *permanent program* for distributing funding to OSC collectives via quadratic funding. Rather than operate a separate platform, base the weighting on monthly recurring donors and donations to a project, and set some basic minimum standards of inclusion: Presumably collectives should be pushing code within some threshold of recent activity, maybe set expectations of participating in OSC communications actively or something too.

Then, instead of setting up for a particular donation drive, folks can encourage funding their own collectives and communicating through those collectives, but programs like Gitcoin Grants can occasionally roll through and disburse some amount of funding to support open source software and it gets distributed to active OSC collectives. Basing the weighting on donors with recurring subscriptions will incentivize stable funding, and help drive funding that happens persistently as opposed to just during specific rounds of activity.

Regarding project exposure, I think OSC could/should expand highlighting projects which are participating and in need of support.

**Reply by gusaus** [reply]
Piggybacking on @ocdtrekkie's great suggestions with some of mine from previous discussions in [#opensource](https://opencollective.slack.com/archives/C0RD4S62E) and [#fundoss](https://opencollective.slack.com/archives/C01PM2BJFQQ).

> Maintaining the separate FundOSS platform is expensive/inefficient use of OSC resources.

A separate platform seems expensive/inefficient, but would it be feasible to incorporate all the features into https://github.com/opencollective/opencollective? What if the functionality discussed in this issue was built into the new site for https://www.oscollective.org/? Apparently, a new site is in the works, but I'm not sure how OSC collects and incorporates feedback from their collectives.

> FundOSS funding participation doesn't roll into OpenCollective funding participation.

Centralizing activity in #opensource, while at the same time leveraging other Slack (#general, #fundraising, #sustainoss, etc.) and marketing channels (Sustain, Gitcoin, etc.) would help solve this problem. We could work with Gitcoin to incorporate additional knowledge and [processes](https://gitcoin.co/fund) (wherever relevant and appropriate) they've accumulated from running successful rounds over time.

> FundOSS contributors don't get updates/communication from the collective. Collectives didn't know if or when future FundOSS rounds would be coming.

Those problems would be addressed by a well-defined process + more engaged community (the latter could be accomplished by enabling a [#community-working-group](https://opencollective.slack.com/archives/C94E046SY) member to complete https://github.com/opencollective/opencollective/issues/1996 with a bounty)

Personally, I'd like to see a combination of official rounds (again, I think we could look at Gitcoin for structure and selection criteria (I 'believe' anyone can opt-in - which seems preferable) + community-organized fundraisers. Again, I think OSC specific fundraisers via https://www.oscollective.org/ would be advantageous - especially if other functionality (like a marketplace/job board and partner program) that helps connect projects to resources were also available.

It seems like [FundOSS](https://opencollective.com/fundoss) (as a [Fund](https://blog.opencollective.com/funds-for-open-source/)) could be better utilized by creating '[projects](https://docs.opencollective.com/help/collectives/projects)' for rounds or campaigns around specific interest areas, categories, or software types. That may encourage large companies in specific interest areas or foundations with industry-specific grants to fund.

Finally - IF cost and team resources (developer, organizer, marketing, etc) were indeed blockers, could we create a 'project' specifically for platform improvements and other resources we'd need? With a budget, the OSC working group could help the OSC team wherever there's a need.

Would be good to have input from other pilot participants and OSC collectives. Would getting this program back on track be beneficial?

**Reply by gusaus** [reply]
I'd like to reference this [relatively current comment](https://github.com/FUNDOSS/fundoss/discussions/44#discussioncomment-7503344) to frame a followup here. 

As noted in this related bit of [Canvas discussion](https://opencollective.slack.com/archives/C05H1N51QR2/p1699466791610049?thread_ts=1698910606.084119&cid=C05H1N51QR2), most of what I was describing in the previous comment accurately describes what we (community members, ecosystem partners, and the originals) could do with a budget.  

Main difference is [#community-working-group](https://opencollective.slack.com/archives/C94E046SY) has made some progress over the past year building capacity and circling back to [enabling and organizing the Open Collective community](https://github.com/orgs/communitycontributorgroup/discussions/2).

While the larger effort has been slow going without support from Open Collective entities, we could make FundOSS a priority if there was a budget to draw from in https://opencollective.com/fundoss

**Comment by gusaus** [comment]
Piggybacking on @ocdtrekkie's great suggestions with some of mine from previous discussions in [#opensource](https://opencollective.slack.com/archives/C0RD4S62E) and [#fundoss](https://opencollective.slack.com/archives/C01PM2BJFQQ).

> Maintaining the separate FundOSS platform is expensive/inefficient use of OSC resources.

A separate platform seems expensive/inefficient, but would it be feasible to incorporate all the features into https://github.com/opencollective/opencollective? What if the functionality discussed in this issue was built into the new site for https://www.oscollective.org/? Apparently, a new site is in the works, but I'm not sure how OSC collects and incorporates feedback from their collectives. 

> FundOSS funding participation doesn't roll into OpenCollective funding participation.

Centralizing activity in #opensource, while at the same time leveraging other Slack (#general, #fundraising, #sustainoss, etc.) and marketing channels (Sustain, Gitcoin, etc.) would help solve this problem. We could work with Gitcoin to incorporate additional knowledge and [processes](https://gitcoin.co/fund) (wherever relevant and appropriate) they've accumulated from running successful rounds over time. 

> FundOSS contributors don't get updates/communication from the collective. Collectives didn't know if or when future FundOSS rounds would be coming.

Those problems would be addressed by a well-defined process + more engaged community (the latter could be accomplished by enabling a [#community-working-group](https://opencollective.slack.com/archives/C94E046SY) member to complete https://github.com/opencollective/opencollective/issues/1996 with a bounty)

Personally, I'd like to see a combination of official rounds (again, I think we could look at Gitcoin for structure and selection criteria (I 'believe' anyone can opt-in - which seems preferable) + community-organized fundraisers. Again, I think OSC specific fundraisers via https://www.oscollective.org/ would be advantageous - especially if other functionality (like a marketplace/job board and partner program) that helps connect projects to resources were also available. 

It seems like [FundOSS](https://opencollective.com/fundoss) (as a [Fund](https://blog.opencollective.com/funds-for-open-source/)) could be better utilized by creating '[projects](https://docs.opencollective.com/help/collectives/projects)' for rounds or campaigns around specific interest areas, categories, or software types. That may encourage large companies in specific interest areas or foundations with industry-specific grants to fund.

Finally - IF cost and team resources (developer, organizer, marketing, etc) were indeed blockers, could we create a 'project' specifically for platform improvements and other resources we'd need? With a budget, the OSC working group could help the OSC team wherever there's a need. 

Would be good to have input from other pilot participants and OSC collectives. Would getting this program back on track be beneficial?

**Comment by gusaus** [comment]
Maybe there was some discussion on one of the monthly OSC calls, but it doesn't seem like there's been any input from OSC about the future of this program (even though folks occasionally inquire in Slack). Now it looks like https://fundoss.org/ is down. Does that mean OSC has officially shelved the program? If yes, could the community help bring it back as suggested in the past couple comments? https://github.com/FUNDOSS/fundoss/issues/42#issuecomment-1216834759 https://github.com/FUNDOSS/fundoss/issues/42#issuecomment-1216999196

**Comment by gusaus** [comment]
Circling back here after coming across what might be a more advanced platform we could leverage for a revival? Not sure if anyone from OSC has talked with @owocki about https://github.com/gitcoinco/skunkworks/issues/284

Would love to touch base and see if there's something we could leverage and contribute to.

**Comment by gusaus** [comment]
Following up on my last comment, I finally was able to connect with @owocki after leaving [this followup](https://community.supermodular.xyz/t/sip-cohort-2-opportunity-3-web2-5-simplegrants-platform/95/2?u=gusaus) in https://community.supermodular.xyz/ about the platform they've been developing. 

> we’ve got a working platform up at [https://simplegrants.xyz/](https://simplegrants.xyz/) but so far do not have a pilot customer / funder for it! we designed it to be well documented and reusable for other purposes too

To which I asked some followup questions:

> So based on your experience and previous involvement with the FundOSS pilot: 1) would you recommend rebuilding on the new platform instead of the existing/broken one? 2) do you have a rough idea of the time/cost it would take to replicate/[improve](https://github.com/FUNDOSS/fundoss/issues/42) the functionality the FundOSS platform offered?; 3) assuming we get buy in from some of the larger groups/collectives in the [Open Collective ecosystem](https://discover.opencollective.com/), are there potential opportunities to partner/pool resources (talent and funds) with y’all?

After mentioning they'd love to support, I sent a follow up email (as requested) to get his recommendations on next steps.

Haven't heard back so I thought I'd just direct everybody back here since we'll need input from @BenJam @zherring and the other original organizers/partners to gauge interest and potential involvement in a FundOSS revival. If for whatever reason they're not interested/able, would it be possible to for new partners and organizers to take over this Github org and https://opencollective.com/fundoss?

**Comment by zherring** [comment]
@gusaus appreciate the tag! I'm currently COO/CPO of a startup and a new dad, so my time is pretty restricted at the moment. Cannot contribute many hours but happy to provide input where it makes sense! Excited to hear folks are wanting to contribute to push this work forward!

**Comment by gusaus** [comment]
As most folks looped into this issue are aware we're in the process of [drafting a proposal](https://docs.google.com/document/d/13Cqc4bhlBZ5q4bqdEhjvq9EnAOXwjpJSrxwg4JBpb6Y/edit?usp=sharing) for the Drupal Association to leverage Open Collective and Quadratic Funding as a way to reach/surpass their fundraising goals (see [this comment for context](https://docs.google.com/document/d/13Cqc4bhlBZ5q4bqdEhjvq9EnAOXwjpJSrxwg4JBpb6Y/edit?disco=AAAAxI23yuU)). 

Since we're essentially proposing to create a flavor of FundOSS (Fund + Quadratic Funding platform) with most of the key FundOSS people/partners already involved (or showing interest), I'd like to suggest (again) that we revisit  https://github.com/FUNDOSS/fundoss/issues/42#issuecomment-1216999196 and other comments and cobble together a plan to revive FundOSS by forking/extending/contributing to https://github.com/supermodularxyz/simplegrants (as opposed to fixing https://github.com/FUNDOSS/fundoss).

As @owocki noted in the Drupal proposal comments...

> @easonchai built simplegrants for us + could probably pitch in with answers about the tech stack + how we designed it. basically we designed it to be easily forkable / skinnable. but fair warning, you all would be the first pilot user of it, so theres likely some rough edges to work out.

Why not gather together all the interested contributors/partners, plan, and pool resources to finish/test a forkable/skinnable MVP?

**Comment by gusaus** [comment]
Had a short Slack convo with @BenJam after sending out the following update about the Drupal proposal and another potential opportunity for a FundOSS round. 

> Quick update via email to say we now have folks at the ED/CEO level for WordPress and Drupal aware and/or reviewing Quadratic Funding proposals. Tim (CTO of the Drupal Association) also talked with Lauren (Ops from Open Source Collective) about certain aspects of our proposal(s) last week.

> While Drupal/WordPress interest has been the catalyst for me suggesting we [revive FundOSS](https://github.com/FUNDOSS/fundoss/issues/42#issuecomment-1630012982), the [recently announced](https://www.drupal.org/association/blog/open-source-unity-joint-concerns-over-the-proposed-cyber-resilience-act-in-the-eu)/[unprecedented collaboration](https://www.youtube.com/watch?v=tbnblHzHqBU) (between [Open Source Matters, Inc./Joomla](https://www.opensourcematters.org/), [Typo3](https://typo3.org/), [WordPress](https://wordpress.com/), and the [Drupal Association](https://www.drupal.org/association)) creates a need/opportunity to run a curated FundOSS AND [Gitcoin Grants](https://grants.gitcoin.co/) rounds to raise awareness and funds for a collaboration between open source projects/communities that power 1/2 of the web.

> Capacity is a non-issue since Drupal and WordPress alone have [global](https://make.wordpress.org/) [communities](https://www.drupal.org/community) and [self-sustaining](https://www.drupal.org/blog/who-sponsors-drupal-development-2020-2021-edition) [economies](https://wpengine.com/wordpress-economy/) that would support everything from developing/maintaining the platforms, improving/extending guides, organizing rounds, fundraising, and more. 

> Because Open Collective would be a requirement (at least for FundOSS and related rounds) most of the participating projects would raise enough funds to raise their profile to funders and contributors in the larger [Open Collective/Open Source ecosystem](https://discover.opencollective.com/?tag=open+source). Participation in [Gitcoin Grants](https://grants.gitcoin.co/) (which most likely would happen independently?) would further diversify and expand the funder and contributor base. 

> Can we bring in some new players and partners with capacity/expertise to build on the foundation that Gitcoin and OSC created (with the FundOSS pilot)? 

Ben mentioned they 'would' support a fundraising effort through the platform after making some of the improvements mentioned in this issue. To which I mentioned @owocki's offer of platform support (see https://github.com/FUNDOSS/fundoss/issues/42#issuecomment-1630012982 and [the related comment thread](https://docs.google.com/document/d/13Cqc4bhlBZ5q4bqdEhjvq9EnAOXwjpJSrxwg4JBpb6Y/edit?disco=AAAA1AGBr2w) in the Drupal proposal).

@BenJam Considering we've essentially accepted that offer and have been planning on bringing on new partners to provide support (as mentioned in my email update), would it make sense to unify our collaboration/partnership around funding experiments as FundOSS but use the new Simplegrants platform?

**Comment by hestenet** [comment]
Chiming in here, as @gusaus has mentinoed me a few times as the CTO over at the Drupal Association. Certainly I'm interested in finding a way to enable the Drupal project to participate in future ventures here, and to cross promote the idea. 

IMO - defining the minimum viable product to launch the next experiment is always the best way forward, and then continuing to iterate. There are a lot of possible avenues as described extensively by @gusaus, and even if we can't achieve all aspects, wherever there is consensus and capacity I'm happy to participate.

**Reply by gusaus** [reply]
@hestenet Using the issue to discussion conversion to provide a more timely/relevant response, based on related projects, goals, and priorities in the Drupal community (also hid my initial followup comment for better flow).

Actually would like to reference the [Leveraging Open Collective and Quadratic Funding for the Drupal Association](https://docs.google.com/document/d/13Cqc4bhlBZ5q4bqdEhjvq9EnAOXwjpJSrxwg4JBpb6Y/edit?usp=sharing) proposal you (in your role as CTO), the DA CEO and @owocki (at the time on behalf of [supermodular.xyz](https://supermodular.xyz/)) have been evaluating, and say this (leveraging the FundOSS platform and program) would eliminate the need for the DA to make any significant time/funding investment [in the platform](https://docs.google.com/document/d/13Cqc4bhlBZ5q4bqdEhjvq9EnAOXwjpJSrxwg4JBpb6Y/edit#bookmark=id.7y1r16wp9is9).

Ideally you (and other partners/contributors for the Drupal project/community) could continue to provide feedback and guidance. Especially since there's a good bit of overlap with projects and integrations that will in-turn be leveraging/extending [Open Collective](https://www.drupal.org/project/opencollective) and potentially [Quadratic Funding](https://www.drupal.org/project/innovation_ideas).

**Comment by gusaus** [comment]
@hestenet Unless I'm misunderstanding @BenJam's email and [Slack](https://opencollective.slack.com/archives/C05H1N51QR2/p1692195597575759?thread_ts=1691498918.310359&cid=C05H1N51QR2) comments, I think we have the green light to revive and extend FundOSS by building features/improvements into the https://github.com/supermodularxyz/simplegrants MVP that @owocki @easonchai are supporting.  

[Community Contributor Group](https://opencollective.com/community-contributor-group) (which includes members connected with larger orgs/communities with capacity) can fill contributor and support needs assuming the new product owners can join us to lead and advise.

**Comment by gusaus** [comment]
> I think we have the green light to revive and extend FundOSS by building features/improvements into the https://github.com/supermodularxyz/simplegrants MVP that @owocki @easonchai are supporting.

Unfortunately we got word from @owocki that @easonchai is no longer available to support simplegrants and we're not in a position to take over w/out any funding/direction/involvement from the product owners or original partners. 

With at least two of the largest open source projects/communities/[economies](https://wpengine.com/resources/value-of-wordpress-worlds-first-study-of-wordpress-economy/)/[digital public goods](https://www.drupal.org/association/blog/drupal-officially-achieves-recognition-as-a-digital-public-good) (and Open Collective [users](https://opencollective.com/automattic-inc)/[investors](https://opencollective.com/dries)) showing interest in participating and a larger role in the ecosystem, I'm hoping the partners and providers see the value of playing some sort of realistic/mutually beneficial role in enabling Quadratic/Democratic Funding (with some of the above fixes/improvements in place) and other [funding experiments](https://opencollective.slack.com/docs/T0HSL38JD/F05H1N51QR2) under the FundOSS umbrella. 

With the product owner and a skilled PM onboard we could probably updating our needs/requirements in https://github.com/supermodularxyz/supermodules/issues/1 and post a RFP or bounty (using the format [similar to Mautic](https://contribute.mautic.org/product-team/request-for-proposals-rfp-process)) as a project on our [Funding Tools collective](https://opencollective.com/funding-tools).

**Comment by gusaus** [comment]
Quick update on the last comment to say that we've picked the discussion with @owocki (primarily in this [Twitter thread](https://twitter.com/gusaus/status/1704571945281536388)) about leveraging/extending the [Simple Grants platform](https://simplegrants.xyz/) he helped get close to MVP https://github.com/supermodularxyz/simplegrants/issues/1 while at https://supermodular.xyz/ for our planned FundOSS comeback (and expansion).

Skim through the Twitter thread and you'll see some productive/promising input from multiple folks at https://www.gitcoin.co/ (the key partner and inspiration for the [FundOSS pilot](https://opencollective.com/fundoss). Main reason for this is Kevin is back at the [company he founded](https://gov.gitcoin.co/t/a-brief-history-of-gitcoin-from-2017-2022/9431) and we're talking about a variety of ways to collaborate and partner. 

Helping get SimpleGrants to a point where we can use for a variety of WordPress, Drupal, and Open Collective ecosystem fundraisers (all leveraging FundOSS and Open Collective) we're planning around [GivingTuesday](https://www.givingtuesday.org/) and the duration of giving season could be an entry point for a subsequent platform and program collaboration and partnership.

To do this, we need to find someone with the following skills (the following is from @easonchai, the former lead dev) along with a product/product manager that could (ideally) work with Kevin to update the scope and estimate the funds needed to finish the job.  

> As Kevin mentioned, a full stack JS-based dev would be great. Thinking more generally, the dev should ideally be quite experienced and versatile, since there are alot of components in the repo itself. For example we not only have the frontend, backend, but also the payment gateway that we have to handle, deployment (so DevOps) and depending on how secure and customized you want the login to be, the authentication can also be expanded and improved on.

>Of course, there are plenty of improvements that can be added on at a later stage in the project, but TLDR:
> - A strong & versatile full stack JS-based dev that can handle the frontend & backend
> - A DevOps engineer to handle deployments & ensuring its scalable & reliable (we don't want downtimes when we play with money!)
> - Someone with experience working with integrating payment gateways like Stripe (just so they know what to do & not to do)

>It can be one dev or two devs, depending on the scope of the project and experience of the devs.

If that's you - or if you know of anyone - please comment here or join [#fundoss](https://opencollective.slack.com/archives/C01PM2BJFQQ) (in Open Collective Slack) and say hello!

**Reply by gusaus** [reply]
I converted this issue to a discussion so it would be easier to drill down on specific aspects referenced in the comments.

Skim through the recent batch and you'll see not only interest from OSC collectives and community members (in regular Quadratic/Democratic Fundraisers) but a few large #opensource projects (like WordPress and Drupal) have been evaluating the model for year end fundraisers in their own (well funded) ecosystems.

As noted in the previous comment, we've been talking to @owocki about leveraging/extending the [Simple Grants platform](https://simplegrants.xyz/) he helped get close to MVP https://github.com/supermodularxyz/simplegrants/issues/1 while at https://supermodular.xyz/ for these fundraisers in the short term. To do this, it would most likely require some degree of involvement/investment from the same folks/orgs that scoped, developed, and maintained the initial platform used for the pilot.  

Just having some combo of @owocki and @BenJam assuming the product owner/manager/advisor role would enable community contributors and partners to handle the rest. Especially if we can post a bounty or participate in a Gitcoin and/or FundOSS QF round to fund the effort!

Would love to hear some thoughts and opinions from the original partners and visionaries!

**Reply by gusaus** [reply]
Quick update to say we tabled our push to have a functional FundOSS platform until admins/partners of the pilot had a chance to revisit and reasses their role, commitment, investment, and benefits in future Democratic/Quadratic Funding rounds. 

While some of the new admins and partners have made progress identifying participants and sponsors (Drupal and WordPress seem very interested in ongoing rounds), we've decided to table the idea of FundOSS or Simplegrants platforms as a stopgap and see if extending/contributing to https://www.gitcoin.co/grants-stack (and reconstituting our partnership with Gitcoin) could be an option in early(ish) 2024.

Still would require some input from @BenJam @owocki and/or whoever they recommend should be driving partnership and product discussions on behalf of Open Collective/OSC and Gitcoin. 

Ideally we'd be able to reconnect after the holidays and see if/how we can go back to the source and create something complimentary to https://impact.gitcoin.co/

**Reply by gusaus** [reply]
Dusting off this thread after realizing @owocki and Gitcoin :raised_hands: have restarted development of https://github.com/gitcoinco/simplegrants.xyz :mega::metal:

Skim back through this thread and discussion for context around how we were planning to use (or at least evaluating) as a replacement for https://github.com/FUNDOSS/fundoss

I haven't talked to Kevin recently (not sure if follow team members in this discussion have) but it seems like contributing to SimpleGrants (and revisiting/reconstituting the partnership with [Gitcoin](https://www.gitcoin.co/)) would enable us to get FundOSS (and some of these [other projects/programs](https://github.com/orgs/FUNDOSS/discussions/44#discussioncomment-7503340)) rolling again :clapper:

Would love to hear thoughts from some of the other folks involved.

**Comment by ocdtrekkie** [comment]
Thanks for your continued work on this. FundOSS was immensely helpful to us, and we are currently trying to raise funds for an important project.

**Reply by gusaus** [reply]
@ocdtrekkie As the only other community/collective benefactor and advocate in this discussion, I appreciate the show of support.

As I told @BenJam and other Open Collective leaders in this related Slack [Canvas](https://opencollective.slack.com/docs/T0HSL38JD/F05H1N51QR2) [discussion](https://opencollective.slack.com/archives/C05H1N51QR2/p1691498918310359), I think having OSC (and ideally other Open Collective entities) involved as a partner, collaborator, or just an enabler would empower orgs and individuals representing in their [#opensource](https://opencollective.slack.com/archives/C0RD4S62E) channel and [ecosystem](https://discover.opencollective.com/?tag=open+source) to support and sustain as a community driven effort.

**Reply by ocdtrekkie** [reply]
Yeah I think OSC's involvement is what makes it trustworthy. I am sort of skeptical when something has a cryptocurrency connection, but if it's something OSC is connected up with, I can trust its reputability.

**Reply by gusaus** [reply]
As someone who was actively recommending (and eventually participating), I heard a good bit of similar concerns. Those went away once the organizers made it clear there was no reliance or dependency. Of course those concerns shifted to the platform and funding model (well documented in this issue/discussion). 

Personally, I was blown away how effective the [QF model](https://www.wtfisqf.com/) was for participants of the FundOSS pilot  (rough edges and all) and subsequent [Gitcoin Grants rounds](https://impact.gitcoin.co/).

A renewed effort that brings together all the original partners, Open Collective entities, and other customers/partners/contributors in the OC ecosystem could transform the entire open source landscape.

**Comment by gusaus** [comment]
@ocdtrekkie As the only other community/collective benefactor and advocate in this discussion, I appreciate the show of support. 

As I told @BenJam and other Open Collective leaders in this related Slack [Canvas](https://opencollective.slack.com/docs/T0HSL38JD/F05H1N51QR2) [discussion](https://opencollective.slack.com/archives/C05H1N51QR2/p1691498918310359), I think having OSC (and ideally other Open Collective entities) involved as a partner, collaborator, or just an enabler would empower orgs and individuals representing in their [#opensource](https://opencollective.slack.com/archives/C0RD4S62E) channel and [ecosystem](https://discover.opencollective.com/?tag=open+source) to support and sustain as a community driven effort.
